### PR TITLE
Update vulnerable dependency versions, refactor React Context use

### PR DESCRIPTION
**Pull request by thaorell**
Update packages to eliminate vulnerabilities

In addition, update component context usage to reflect latest React context API [link](https://reactjs.org/docs/context.html)

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by sjd78** [comment]
ci add to white list

**Comment by sjd78** [comment]
ci add to whitelist

File: package.json [hunk 1]
```diff
@@ -78,19 +78,19 @@
     "history": "4.7.2",
     "immutable": "3.8.2",
     "intl-messageformat": "2.2.0",
-    "jquery": "=3.4.1",
-    "lodash": "4.17.11",
+    "jquery": "^3.4.1",
```
Comment (bond95) [inline package.json]: Please, remove `^` sign, and at the rest of file as well.

**Comment by sjd78** [comment]
@thaorell - Is there any issues with upgrading from React 16.4 to 16.8.6?  Any breaking changes or things in upgrade guides?

**Comment by sjd78** [comment]
ci test please

**Pull request merged**